bitcoin/src/blockdata/transaction.rs

```diff
@@ -790,7 +790,7 @@ pub fn effective_value(
     value: Amount,
 ) -> Option<SignedAmount> {
     let weight = satisfaction_weight.checked_add(TX_IN_BASE_WEIGHT)?;
-    let signed_input_fee = fee_rate.checked_mul_by_weight(weight)?.to_signed();
+    let signed_input_fee = fee_rate.fee_wu(weight)?.to_signed();
```

Comment:
What is `fee_wu`? Fee is not measured by weight units, there was some significant misnaming.